services/credential-server-ui/src/pages/CredentialPage.tsx

```diff
@@ -108,30 +110,20 @@ const CredentialPage: React.FC = () => {
           <Grid container spacing={2} justifyContent="center">
             <Grid item xs={10}>
               <FormControl fullWidth>
-                <InputLabel id="selectedContact">
-                  Established Connections
-                </InputLabel>
-
                 <Controller
                   name="selectedContact"
                   control={control}
                   render={({ field }) => (
-                    <Select
-                      labelId="selectedContact"
-                      label="Established Connections"
+                    <Autocomplete
```

Comment:
Can we update the placeholder text to say "Search by connection name or ID..." - I didn't actually realise this was an autocomplete field until seeing the code!